src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBus.java

```diff
@@ -415,19 +415,19 @@ public void addHvdc(LfHvdc hvdc) {
         hvdcs.add(Objects.requireNonNull(hvdc));
     }
 
-    private static double dispatchQ(List<LfGenerator> generatorsThatControlVoltage, boolean reactiveLimits, double qToDispatch) {
+    protected static double dispatchQ(List<LfGenerator> generatorsThatControlVoltage, boolean reactiveLimits, double qToDispatch) {
         double residueQ = 0;
         double calculatedQ = qToDispatch / generatorsThatControlVoltage.size();
```

Comment:
Here it should be important to exclude generatorsThatControlVoltage.size() == 0

Reply:
it is handled by the method calling it : 
while (!generatorsThatControlVoltage.isEmpty() && Math.abs(qToDispatch) > Q_DISPATCH_EPSILON) {
            qToDispatch = dispatchQ(generatorsThatControlVoltage, reactiveLimits, qToDispatch);
        }

Reply:
but I can add a check

Reply:
It should not happen so I added a exception